### ISSUE TITLE
Pack type weaving chart

### ISSUE THREAD

**Issue by CableGuy67** [Shelagh-Lewins/twisted-threads-2#5]
In the same way that we can save for printing a pattern using the "Printer Friendly Pattern" it would be nice to have a pack notation as well. Something like is used in Linda Hendrickson's books. I use a modified version of this that is a little more flexible since hers is really designed as a pattern pack and a background pack.
Simply it is just a pick number followed by a number denoting the number of tablets that will turn the same way, followed by a number that will turn the opposite way, back and forth until all the tablets are noted. This is to make it easy to slide the tablets either to the Toward or Away packs. I suppose one could also just turn them in place (count first number of tablets and turn them, the next number turn opposite) instead of counting them out.
I'll attach a photo of my method for further discussion when the time comes.
![helix normal](https://user-images.githubusercontent.com/504816/76309617-cae85000-62d5-11ea-9c69-2e8cc958e3f9.jpg)

**Comment by Shelagh-Lewins** [comment]
A related request has been made for a chart showing which cards are turning forwards / backwards per pick.
![Weaving chart suggestion](https://github.com/Shelagh-Lewins/twisted-threads-2/assets/15693742/2fbef7c7-5153-4675-b792-9e30fc7759b2)

**Comment by CableGuy67** [comment]
Yeah, I have used the second method as well since so many patterns do use it. I like it when you have to do multiple picks that are the same and just make a forward and backward set of tablets.
A lot of times when I use the notation from my first picture it is because I'm just walking through the tablets without making a forward and back pack. I have a little 3D printed thing where I can see one pick in a slot to keep track of things.

It's nice to see things going on here. I've been doing too much other stuff lately and haven't done much weaving but that always seems to change when I need new bands to make bags. :)
